Process HTEX-Worker-1:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-3:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-0:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-4:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-2:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-5:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-7:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-6:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-8:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-9:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-10:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-11:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-13:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-14:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-12:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-15:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-17:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-16:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-18:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-19:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-20:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-21:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-22:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-23:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-24:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-25:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-26:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-27:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-28:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-30:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-29:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-31:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-33:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-34:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-36:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
Process HTEX-Worker-37:
Traceback (most recent call last):
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/python/3.9.16/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/lib/python3.9/site-packages/parsl/process_loggers.py", line 26, in wrapped
    r = func(*args, **kwargs)
  File "/scratch/users/pdanie20/PD-stopping-power-ml/venv/bin/process_worker_pool.py", line 685, in worker
    assert cores_per_worker > 0, "Affinity does not work if there are more workers than cores"
AssertionError: Affinity does not work if there are more workers than cores
/usr/local/python/3.9.16/lib/python3.9/multiprocessing/resource_tracker.py:216: UserWarning: resource_tracker: There appear to be 7 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '
